multiline/parsers_multiline.conf

```diff
@@ -0,0 +1,16 @@
+[MULTILINE_PARSER]
+    name          multiline-regex-test
+    type          regex
+    # Regex rules for multiline parsing
+    # ---------------------------------
+    #
+    # configuration hints:
+    #
+    #  - first state always has the name: start_state
+    #  - every field in the rule must be inside double quotes
+    #
+    # rules   |   state name   | regex pattern                   | next state name
+    # --------|----------------|--------------------------------------------------
+    rule         "start_state"   "/(Dec \d+ \d+\:\d+\:\d+)(.*)/"  "cont"
+    rule         "cont"          "/^\s+at.*/"                     "cont"
```

Comment:
I am not sure but the state_name and next_state_name are same. Is that OK?

Reply:
yeah It works. It is from upstream doc